test/feature/features/coordinator.feature

```diff
@@ -0,0 +1,235 @@
+Feature: Coordinator test
+  Background:
+    Given cluster is up and running
+    When I run SQL on host "coordinator"
+    """
+    REGISTER ROUTER r1 ADDRESS regress_router::7000
+    """
+    Then command return code should be "0"
+
+    When I run SQL on host "coordinator"
+    """
+    CREATE SHARDING RULE r1 COLUMN id;
+    CREATE KEY RANGE krid1 FROM 0 TO 10 ROUTE TO sh1;
+    CREATE KEY RANGE krid2 FROM 11 TO 30 ROUTE TO sh2;
+    """
+    Then command return code should be "0"
+
+    When I run SQL on host "router"
+    """
+    CREATE TABLE test(id int, name text)
+    """
+    Then command return code should be "0"
+
+  Scenario: Add/Remove router
+    When I run SQL on host "coordinator"
+    """
+    UNREGISTER ROUTER r1
+    """
+    Then command return code should be "0"
+
+    When I run SQL on host "coordinator"
+    """
+    SHOW routers
+    """
+    Then SQL result should match regexp
+    """
+    \[\]
+    """
+
+    When I run SQL on host "coordinator"
+    """
+    REGISTER ROUTER r2 ADDRESS regress_router::7000;
```

Comment:
What happen if you register two routers with the same address?

Reply:
I'll extend the scenario and push commit